common/src/main/java/io/quarkus/ts/openshift/common/Command.java

```diff
@@ -1,18 +1,24 @@
 package io.quarkus.ts.openshift.common;
 
+import static org.fusesource.jansi.Ansi.ansi;
+
 import java.io.BufferedReader;
 import java.io.File;
+import java.io.FileOutputStream;
 import java.io.IOException;
+import java.io.InputStream;
 import java.io.InputStreamReader;
+import java.io.OutputStream;
 import java.util.Arrays;
 import java.util.List;
-
-import static org.fusesource.jansi.Ansi.ansi;
+import java.util.function.BiConsumer;
```

Comment:
I used the same formatter than Quarkus ... should I revert this and use another? Don't want to create unnecessary changes everywhere.

Reply:
There's an import sort Maven plugin configured in this project, `mvn compile` should apply I think?

Reply:
I don't mean only about sort import, but general formatter rules. This is the formatter I think we should follow: https://github.com/quarkusio/quarkus/tree/master/independent-projects/ide-config/src/main/resources

Reply:
yup, formater inclusion is on our todo, last time @Ladicek had some objections to Quarkus formatter, so we didn't push hard on it. But as more people will be looking into the project, it might be time to push it :)

Reply:
The Eclipse formatter is crap and more often than not, I have to fight it to leave the code at least a little readable. I hate it with a passion, but feel free to use it on new code :-)

Reply:
Whole codebase would get (re)formatted, I would go with https://github.com/quarkusio/quarkus/blob/master/build-parent/pom.xml#L459 way not to need to worry about everyone's IDE setup